heidrun/mdl_map.rb

```diff
@@ -0,0 +1,77 @@
+Krikri::Mapper.define(:mdl_map, :parser => Krikri::JsonParser) do
```

Comment:
That's a good idea, but I have no idea how to start figuring out how to do that.  Could someone point me in the right direction?